cli/formatter/format_test.go

```diff
@@ -311,6 +311,15 @@ func TestFormatter_MakeOutputFormat(t *testing.T) {
 			"",
 			true,
 		},
+		{
+			// when user typed to console:
+			// localhost:xxxx> \set output table
+			// localhost:xxxx> {}
+			formatter.TableFormat,
+			"{}",
+			"+------+\n| col1 |\n+------+\n|      |\n+------+\n",
```

Comment:
To be honest, this doesn't look like a correct output. There shouldn't be any table here.